src/activity_actor.cpp

```diff
@@ -2139,13 +2140,77 @@ std::unique_ptr<activity_actor> stash_activity_actor::deserialize( JsonIn &jsin
     return actor.clone();
 }
 
+void bookbinder_copy_activity_actor::start( player_activity &act, Character & )
+{
+    act.moves_total = to_moves<int>( pages * 2_minutes );
+    act.moves_left = to_moves<int>( pages * 2_minutes );
```

Comment:
2 minutes per page seems quite rapid to me. I don't have any concrete suggestions on what it _should_ be, this just seems quite fast, and maybe someone else will.

Reply:
Maybe 8 ~ 10 minutes per page? In which case a way to resume the activity is definitely needed.

Reply:
I'd err towards 10 minutes, but that works.

Reply:
I took some random cooking recipes from a website and tried rewriting them on paper and got following times per pages (A4, 28 lines per page on average):
5:30.42
6:13.90
4:56.59
5:25.45
8:03.70

So median is ~6 minutes, with maximum at 8.

Reply:
What level skill would they correspond to? I'm caring more about the use case of implied recipes (which cooking recipes definitely are not qualifying as, at least where you're demonstrating), where's it's either a mention you can do something, or a poorly written recipe, or just something you figure out from the information in the book. Given those values, I think this is a reasonable compromise.